client/related_items/helpers.js

```diff
@@ -79,21 +79,37 @@ const findComparisonFeatures = (mainFeatures, comparisonFeatures) => {
   let allFeatures = [];
 
   mainFeatures.forEach(feature => {
-    feature.compValue = '';
+    if (feature.value === null) { feature.value = '--'; }
+    feature.compValue = '--';
+    feature.product = 'main';
     allFeatures.push(feature);
   });
 
   comparisonFeatures.forEach(compFeature => {
+    if (compFeature.value === null) { compFeature.value = '--'; }
     var index = allFeatures.findIndex(item => item.feature === compFeature.feature);
+
     if (index !== -1) {
       allFeatures[index].compValue = compFeature.value;
+      compFeature.product = 'comp';
     } else {
-      compFeature.compValue = '';
+      compFeature.compValue = '--';
+      compFeature.product = 'comp';
       allFeatures.push(compFeature);
     }
   });
 
   return allFeatures;
 };
 
-module.exports = { getRelatedItems, getProductInfo, getProductStyles, getProductRatings, findDefaultStyle, defineMainProduct, findComparisonFeatures };
\ No newline at end of file
+const preventScroll = () => {
+  let elem = document.getElementsByTagName('body')[0];
+
+  if (elem.style.overflow === '') {
```

Comment:
I trust that this works, but is the default value for over flow not 'visible'? Just curious.

Reply:
Good question - not sure! I may try it and push an update in a later PR if that works.